src/Symfony/Component/Validator/ConstraintViolation.php

```diff
@@ -79,13 +79,13 @@ public function __toString()
         }
 
         $propertyPath = (string) $this->propertyPath;
-        $code = $this->code;
+        $code = trim($this->code);
```

Comment:
trimming is a new behavior, which IMHO is not needed.

Reply:
I understand your point, and I agree. My concern was originated by the absence of delimiters around the printed code, so I thought that readability could be improved trimming any undesired blankspace around the code.

Reply:
the delimiter is `(code ` and `)` :)

> undesired blankspace

we don't know that

Reply:
I mean cases like
```
(code    )
```
I think it is not clear which space comes from the dumped message and which one comes from the actual code, but I'm pretty sure this is an edge case that we can ignore.